python/langsmith/testing/_internal.py

```diff
@@ -1302,3 +1303,25 @@ def _stringify(x: Any) -> str:
         return dumps_json(x).decode("utf-8", errors="surrogateescape")
     except Exception:
         return str(x)
+
+
+def _dumpd(x: Any) -> Any:
+    """Serialize LangChain Serializable objects."""
```

Comment:
I think we should try to at least replace other non-serializable objects rather than just failing

Reply:
like pydantic?

Reply:
Yeah or other things

Reply:
It also doesn't handle arrays well, error message is confusing:

`dictionary update sequence element #0 has length 7; 2 is required`

But would be down to just ship this first if it'll fix messages in general